app/models/declaration/submissions/SubmissionStatus.scala

```diff
@@ -25,16 +25,14 @@ object SubmissionStatus extends Enumeration {
   implicit val format: Format[SubmissionStatus.Value] = EnumJson.format(SubmissionStatus)
 
   val PENDING, REQUESTED_CANCELLATION, ACCEPTED, RECEIVED, REJECTED, UNDERGOING_PHYSICAL_CHECK, ADDITIONAL_DOCUMENTS_REQUIRED, AMENDED, RELEASED,
-    CLEARED, CANCELLED, CUSTOMS_POSITION_GRANTED, CUSTOMS_POSITION_DENIED, GOODS_HAVE_EXITED_THE_COMMUNITY, DECLARATION_HANDLED_EXTERNALLY,
```

Comment:
Why have you removed `GOODS_HAVE_EXITED_THE_COMMUNITY` sub status?

Reply:
Never  used

Reply:
Yes please. It is certainly used on the BE https://github.com/hmrc/customs-declare-exports/blob/ad2daf7d76c82ad14de2f44ecdbb1c8e2036803e/app/uk/gov/hmrc/exports/models/declaration/submissions/SubmissionStatus.scala#L47. Need to keep FE and BE in sync